src/styles/index.scss

```diff
@@ -1,3 +1,115 @@
+html {
+  font-family: Roboto, sans-serif;
+}
+
 body {
   margin: 0;
 }
+
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 198px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  background: #fff;
+
+  &__img {
+    width: 160px;
+    height: 134px;
+    margin: 32px 19px 0;
+  }
+
+  &__title {
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    margin: 40px 16px 0;
+    color: #060b35;
+  }
+
+  &__code {
+    font-size: 10px;
+    line-height: 14px;
+    font-weight: 400;
+    margin: 4px 84px 0 16px;
+    color: #616070;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin: 24px 16px 0;
+
+    &-text {
+      margin: 0;
+      font-size: 12px;
+      line-height: 18px;
+      color: #616070;
+    }
+
+    &-amount {
+      margin: 0;
+      font-weight: 700;
+      font-size: 16px;
+      line-height: 18px;
+      color: #060b35;
+    }
+  }
+
+  &__button {
+    box-sizing: border-box;
+    text-decoration: none;
+    width: 166px;
+    height: 40px;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    font-weight: 700;
+    font-size: 14px;
+    line-height: 16px;
+    border-radius: 5px;
+    background-color: #00acdc;
+    color: #fff;
+    margin: 16px;
```

Comment:
preferably to sort your css styles (everywhere)
for example:
- position
- display
- margin / padding
- font settings
- colors